[SafePlaces] Read spjson files to GeoJSON

vladgp opened the pull request:
Using the "Custom Map Data" dialog, can load a file that looks like:

```
$ head -10 ../output.spjson
[
  {"time": 1583084539712, "lat": 42.3962436, "lon": -71.0998953 },
  {"time": 1583085476383, "lat": 42.3946736, "lon": -71.0823379 },
  {"time": 1583085476383, "lat": 42.3936728, "lon": -71.080277 },
  {"time": 1583085992817, "lat": 42.3943908, "lon": -71.0829812 },
  {"time": 1583087568000, "lat": 42.4340804, "lon": -71.2341198 },
  {"time": 1583087568000, "lat": 42.4359938, "lon": -71.2342089 },
  {"time": 1583089157000, "lat": 42.4360062, "lon": -71.2346891 },
  {"time": 1583089737000, "lat": 42.4707332, "lon": -71.2098715 },
  {"time": 1583089737000, "lat": 42.470407, "lon": -71.2083229 },
...
]
```

<img width="1920" alt="Screen Shot 2020-03-15 at 11 29 52 PM" src="https://user-images.githubusercontent.com/2821585/76721491-a39ae400-6716-11ea-8939-9cdd98dc63f9.png">

Bonkles commented on modules/svg/data.js:
```diff
@@ -314,6 +314,27 @@ export function svgData(projection, context, dispatch) {
         return (new DOMParser()).parseFromString(textdata, 'text/xml');
     }
 
```
There's a convention within iD of leaving two spaces between function calls, so please add another space here!

vladgp replied:
Done!

vladgp commented:
@Bonkles - I addressed all the issues `npm test` found in `data.js`, and added an extra newline per your comment. `npm test` still fails for me with the following:

```
Error: Could not resolve '../util' from ../data.js
    at error (/Users/vgp/work/privatekit/RapiD/node_modules/rollup/dist/rollup.js:5365:30)
    at ModuleLoader.handleMissingImports (/Users/vgp/work/privatekit/RapiD/node_modules/rollup/dist/rollup.js:12249:17)
    at ModuleLoader.<anonymous> (/Users/vgp/work/privatekit/RapiD/node_modules/rollup/dist/rollup.js:12140:30)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/vgp/work/privatekit/RapiD/node_modules/rollup/dist/rollup.js:41:28) {
```
